substrate/frame/support/procedural/tools/src/lib.rs

```diff
@@ -114,7 +117,9 @@ pub fn generate_hidden_includes(unique_id: &str, def_crate: &str) -> TokenStream
 /// Generates the path to the frame crate deps.
 fn get_frame_crate_path(def_crate: &str) -> Option<syn::Path> {
 	// This does not work if the frame crate is renamed.
-	if let Ok(FoundCrate::Name(name)) = crate_name(&"frame") {
+	if let Ok(FoundCrate::Name(name)) =
+		crate_name(&"polkadot-sdk-frame").map_err(|_| crate_name(&"frame"))
```

Comment:
```suggestion
		crate_name(&"polkadot-sdk-frame").or_else(|_| crate_name(&"frame"))
```

not sure about the return type of `crate_name` but using `or_else` seems correct to me here.